k3d/plot.py

```diff
@@ -1,14 +1,13 @@
 from __future__ import print_function
 
-from functools import wraps
 import base64
-
 import ipywidgets as widgets
-from traitlets import Unicode, Bool, Int, List, Float
 from IPython.display import display
+from functools import wraps
+from traitlets import Unicode, Bool, Int, List, Float
```

Comment:
Was it sorted by some IDE?

I actually tend to sort them in a following, PEP-8 inspired, fashion:

1. from __future__ imports, alphabetically (otherwise they don't work anyway)
(every block should be sorted alphabetically, anyway)
2. from standard_library import ...
3. import standard_library_module
4. from other_used_library import ...
5. import other_used_module (numpy, IPython, ...)
6. from .our_own_modules import ...
7. import .other_own.module